Hippo.Web/ClientApp/src/components/Order/OrderForm.tsx

```diff
@@ -18,148 +19,130 @@ const OrderForm: React.FC<OrderFormProps> = ({
   readOnly,
   onSubmit,
 }) => {
+  const methods = useForm<OrderModel>({ defaultValues: orderProp });
```

Comment:
doesn't really matter but why make this a two step assignment then destructure operation?

Reply:
@srkirkland because i want to spread all the methods as props to `<FormProvider>` but also select out those properties to use elsewhere. i could've done `methods.submitForm()` but destructuring 4 eva